spec/spec_helper_methods.rb

```diff
@@ -13,7 +13,7 @@ def baseline_os_hash
       },
       {
         'operatingsystem' => 'Ubuntu',
-        'operatingsystemrelease' => %w[20.04 22.04]
+        'operatingsystemrelease' => %w[20.04 22.04 24.04]
```

Comment:
I think in another PR we should remove the whole hash here. It doesn't make sense to duplicate the information from metadata.json. We did that in the past because we only had limited CI power.

Reply:
yes but we need to remove not supported os from metadata.json
I'm waiting centos7 EOL to think to release v11

Reply:
FYI: With https://github.com/voxpupuli/puppet-zabbix/pull/967 the unit tests will run much faster, so we don't need to worry about the CI runtime for unit tests.